### PR TITLE
We THEM GIRLZ 99% Completion About-Page :P

### PR DESCRIPTION
**Pull request by lilcheerio**
Welcome back guys! It's your girls KMS (kilometer squad). 
We're almost done our about page, however we have a media query mishap that needs tended to in the future. It all was working good up to until 1226px to 768px, needs another adjustment for media query here. In this sweet spot, header and content text seems to go off screen above hero image. Once this is complete, he only other things needed to be completely finished would be; header and footer merge, as well as correct working links for them all including our one single button in mobile.

You know you love us!
xoxo gossip girl

### CODE REVIEW

File: css/about.css [hunk 1]
```diff
@@ -15,6 +15,51 @@ Styles for an individual page
...
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	/* max-height: 120px; */
+}
+
+.hero-text {
+	border: 2px solid transparent;
+	width: 369px;
+	/* display: flex;
```
Comment (maxharrisnet) [inline css/about.css]: Please delete comments if this is not being used

File: css/about.css [hunk 2]
```diff
@@ -15,6 +15,51 @@ Styles for an individual page
...
+	color: var(--color-onyx);
+	text-align: center;
+}
+
+h3 {
+	font-family: var(--main-font);
+	font-size: var(--font36);
+	font-weight: var(--font-weight-bold);
+	line-height: var(--line-height40);
+	margin-bottom: 20px;
```
Comment (maxharrisnet) [inline css/about.css:48]: Please use variables where possible

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
This looks good, just a couple of small fixes please

File: css/about.css (hunk 1 above)
Comment (CosmicStarfire) [reply css/about.css]: on it!

File: css/about.css (hunk 2 above)
Comment (CosmicStarfire) [reply css/about.css:48]: on it!

Comment (CosmicStarfire) [reply css/about.css:48]: completed task

File: css/about.css (hunk 1 above)
Comment (CosmicStarfire) [reply css/about.css]: completed task

File: about.html
Comment (maxharrisnet) [inline about.html:1]: Please include the footer. You can copy it from index.html. I think were are still waiting for the header.

**Review by maxharrisnet — CHANGES_REQUESTED** [review]

**Review by maxharrisnet — APPROVED** [review]
This looks excellent - thanks for all of your hard work! 🙌🏾

**Review by maxharrisnet — APPROVED** [review]
I just restored the about.css file that I blew out. The issue was that the about.css file was deleted in a different branch. I am pretty sure I am the one who did this because I saw about.css changes in a branch where they did not belong. There was probably a better fix for that, but that’s Git for you.

When I merged development into about-page (to fix a merge conflict), the about.css deletion came in as well. I am not sure why that did not create another merge conflict…but that’s Git for you :upside_down_face:

Fortunately, you can always press rewind in Git. The solution here was to
do a git log which gives you a list of all of your previous commits
find the last good commit before I did the merge and copied the commit id (977678b02c5a3a74cef194a154c21d2af7e39e84)
I then did git checkout 977678b02c5a3a74cef194a154c21d2af7e39e84 — css/about.css to get the single file from that commit
Commit and push as usual 
Fixed!

Screen Shot 2024-01-22 at 10 27 34 AM

**Review by SolivanLau — APPROVED** [review]
Thank you! good work through styles. and thank you @maxharrisnet for resolving git issues removing about.css!

**Pull request merged**